src/app/campaign.model.ts

```diff
@@ -42,13 +46,17 @@ export class Campaign {
     public target: number,
```

Comment:
does target need to become nullable at this point if a campaign that uses shared funds doesn't have a target of its own? Or even better following on from https://github.com/thebiggive/donate-frontend/pull/1289/files#r1326224857 we could use a union type to say that a campaign either has its own target or it has parent campaign stuff.

Reply:
I think that the reality in the data is that the children still have & surface a "target", which we can see being used on a [Staging campaign of this type](https://donate-staging.thebiggivetest.org.uk/global-2) now – it's just not very meaningful.

I haven't added conditionals that would slow down Salesforce so that will remain the case.

Reply:
OK - maybe worth making it nullable here anyway so that SF is free to stop sending it in future in cases where it isn't meaningful, and to remind us working in frontend to not expect to always be able to (meaningfully) find a target on a campaign object?

Reply:
Makes sense – done on 6f1b592c3939144b760be2a32bc2a788410fa183